### PR TITLE
Feature/664 search improvements - part 1

### PR DESCRIPTION
**Pull request by lraivio**
This PR:
- Adds new search box.
- Keeps search filters on one line.
- Refactors search results and map toggle.

Refs #664.

### CODE REVIEW

File: app/pages/search/results/_search-results.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    border-radius: 100px;
+    padding-left: 40px;
+    padding-right: 40px;
+    font-size: 1.3rem;
+
+    // position: fixed;
+    // bottom: 0px;
+    // width: 800px;
+    // z-index: 100;
+    // border-radius: 0;
```
Comment (YoYuUm) [inline app/pages/search/results/_search-results.scss]: - [ ] Remove commented lines

**Comment by YoYuUm** [comment]
There is going to be conflicts with translation files.

**Comment by YoYuUm** [comment]
Have you consider about reducing the search bar height? I think it takes a lot of space which would be nice for the map.

**Comment by lraivio** [comment]
Fixes made.
I created issue #670 for making "everything" (including the search bar) smaller. That is something we can tweak with Sami so everything looks good and has enough space.

**Pull request merged**